hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/VolumeArgs.java

```diff
@@ -33,25 +33,30 @@
 
   private final String admin;
   private final String owner;
-  private final String quota;
+  private final String storagespaceQuota;
+  private final long namespaceQuota;
   private final List<OzoneAcl> acls;
   private Map<String, String> metadata;
 
   /**
    * Private constructor, constructed via builder.
    * @param admin Administrator's name.
    * @param owner Volume owner's name
-   * @param quota Volume Quota.
+   * @param storagespaceQuota Volume Quota.
+   * @param namespaceQuota Volume Quota.
```

Comment:
I think we could update the description of the parameter consistent with shell-command IMHO.